lib/catalog/martyrology.ts

```diff
@@ -3326,6 +3610,19 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
     passion_of_saint_john_the_baptist: {
       name: 'Passion of Saint John the Baptist',
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-pair-eveque-davranches-et-saint-scubillion-abbe
+    // - https://fr.wikipedia.org/wiki/Paterne_d%27Avranches
+    paterne_of_avranches_bishop: {
```

Comment:
Remember to update also:

- `paterne_of_avranches_bishop_and_scubilion_abbot`;
- `name` (two lines below).

```suggestion
    paternus_of_avranches_bishop: {
```